i18n/translations/en.ts

```diff
@@ -340,12 +340,18 @@ export const en = {
     title: "New group",
   },
 
-  new_conversation: {
+  new_chat: {
     create_group: "Create group",
     back: "Back",
     create: "Create",
     add_members: "Add members",
-    new_conversation: "New conversation",
+    new_chat: "New chat",
+    invite_to_converse: "Invite them to Converse",
```

Comment:
curious, should we do this instead so it's easier to search?
```suggestion
    "invite them to Converse": "Invite them to Converse",
```

Reply:
Love that idea - let's do it eventually

Reply:
tabling for now though